index.html

```diff
@@ -0,0 +1,107 @@
+          <li class="links"><a class="alinks borderLink" href="#">Come Celebrate</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <section class="headContent">
+    <h2 class="headerTitle">"Have a Happy Holi!"</h2>
+    <h1 class="subHeader">HOLI - "Festival of Colors" or the "Festival of Love"</h1>
+    <p class="headerParagraph">Holi has religious and mythological significance, marking the victory of good over evil and the arrival of spring. Holi is now celebrated globally by people of Indian origin and has become a symbol of India's cultural richness and diversity.</p>
+    <p class="headerDate">2023.03.07 ~ 24</p>
+    <p class="headerLocation">@ Vrindavan, INDIA</p>
+    </section>
+  </header>
+
+  <section class="main">
+    <h2 class="sub">Main Program</h2>
+    <p class="line"></p>
+    <div class="mainDiv">
+      <div>
+        <i class="bi bi-flower1 icon"></i>
+        <span>Flower</span>
+        <p>Holi is celebrated on the full moon day in the Hindu month of Phalguna, which falls in February or March.</p>
+      </div>
+      <div>
+        <i class="bi bi-heart-fill icon"></i>
+        <span>Love</span>
+        <p>Holi is also associated with several legends, such as the Love story of Lord Krishna and Radha.</p>
+      </div>
+      <div>
+        <i class="bi bi-paint-bucket icon"></i>
+        <span>Color</span>
+        <p>People throw colored powder and water on each other, sing and dance to traditional music, and feast on sweet delicacies.</p>
+      </div>
+      <div>
+        <i class="bi bi-globe icon"></i>
+        <span>Message</span>
+        <p>Holi is considered one of the major festivals in Hinduism and is celebrated with great fervor and enthusiasm.</p>
+      </div>
+      <div>
+        <i class="bi bi-stars icon"></i>
+        <span>Worship</span>
+        <p>The festival has a cultural significance, bringing people together and fostering unity and happiness.</p>
+      </div>
+    </div>
+    <button class="mainButton">Join Holi 2023</button>
+    <p class="mainP">SEE THE WHOLE PROGRAM</p>
+  </section>
+
+  <!-- Implement the section "Featured speakers" with dynamic HTML. -->
+  <section id="feature" class="feature">
+  </section>
+
+  <section class="sponsor">
+    <h2 class="sub">Partner</h2>
+    <div class="line"></div>
+    <div class="company">
+      <img class="companyLogo" src="./images/mozilla.png" alt="mozilla">
+      <img class="companyLogo" src="./images/google.png" alt="google">
+      <img class="companyLogo" src="./images/naver.png" alt="naver">
+      <img class="companyLogo" src="./images/daumkakao.png" alt="daumkakao">
+      <img class="companyLogo" src="./images/airbnb.png" alt="airbnb">
+    </div>
+  </section>
+
+  <section class="footer">
+    <img class="footImg" src="./images/Logo.png" alt="diwali">
+    <div>
+      <p>2023 Holi India.</p>
+      <p>Some Rights Reserved.</p>
+    </div>
+  </section>
+
+  <script src="main.js"></script>
+</body>
```

Comment:
- [ ] For semantic purposes and for your website to be easily understood by screen readers and properly interpreted by search engines, kindly use` <footer>` tag to layout your footer contents 💯 because footer tag is used for depict the footer of a webpage 👍🏽

Reply:
Thank you @vickymarz  I fixed it

Reply:
@vickymarz Thank you, I fixed it